### PR TITLE
DHFPROD-6301: View JSON and XML documents in Record View when navigated from All Data view

### PR DESCRIPTION
**Pull request by timur-isangulov**
DHFPROD-6302: View JSON and XML documents in Record View when navigated from Entity Specific View.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Review by wooldridge — CHANGES_REQUESTED** [review]
@timur-isangulov, minor consistency question, I'm seeing a "Back to results" link for All Data/Staging but a "Back" link from All Entities/Final. Should these labels be the same and, if so, should this be fixed in this PR?

**Comment by timur-isangulov** [comment]
> Back to results

Thanks @wooldridge, I confirmed that with UX and made them the same in both places "Back to results".

File: marklogic-data-hub-central/ui/e2e/cypress/support/pages/detail.tsx [hunk 1]
```diff
@@ -37,11 +37,11 @@ class DetailPage {
   }
 
   getDocumentJSON() {
-    return cy.get(".pretty-json-container");
+    return cy.findByTestId("json-container");
   }
 
   getDocumentXML() {
-    return cy.get("[data-cy=xml-document]");
+    return cy.findByTestId("xml-container");
```
Comment (Sanjeevani19) [inline marklogic-data-hub-central/ui/e2e/cypress/support/pages/detail.tsx:44]: @timur-isangulov I see getDocumentJSON and getDocumentXML methods are using the same test id's in two different files, Any reason for not combining them together ?

Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/e2e/cypress/support/pages/detail.tsx:44]: Not sure why there were a duplicates but I combined them. Thanks.

**Review by wooldridge — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Review by xnikhil08 — APPROVED** [review]

**Pull request merged**